OracleUnifiedDirectory/README.md

```diff
@@ -0,0 +1,134 @@
+Oracle Unified Directory (OUD) on Docker
+========================================
+
+## Contents
+
+1. [Introduction](#introduction)
+2. [Prerequisites for OUD on Docker](#prerequisites-for-oud-on-docker)
+3. [Prerequisites for OUD on Kubernetes](#prerequisites-for-oud-on-kubernetes)
+4. [Installing the OUD Docker Image](#installing-the-oud-docker-image)
+5. [Running the OUD Docker Image in a Docker Container](#running-the-oud-docker-image-in-a-docker-container)
+6. [OUD Docker Container Configuration](#oud-docker-container-configuration)
+7. [OUD Kubernetes Configuration](#oud-kubernetes-configuration)
+
+## Introduction
+
+Oracle Unified Directory provides comprehensive Directory Solution for robust Identity Management.
```

Comment:
Neither "directory solution" nor "identity management" should be capitalised.

Reply:
Updated